undate/undate.py

```diff
@@ -1,6 +1,9 @@
 import datetime
 from calendar import monthrange
 
+# Pre 3.10 requires Union for multiple types, e.g. Union[int, None] instead of int | None
```

Comment:
I meant to comment and this forgot — thanks for this note, super helpful to have here